tests/data/demo/v1/data_model_configuration.yaml

```
@@ -0,0 +1,86 @@
+generator:
+  production_min:
+    source_file: "files/model.yaml"
+    extraction_path: "model.generator.[name].p_min"
+    is_list: False
+    default_value: 0.0
+    cast_type: float
+  penstock_number:
+    source_file: "files/model.yaml"
+    extraction_path: "model.generator.[name].penstock"
+    is_list: False
+    default_value: 0 # TODO: Check if this is correct
+    cast_type: int
+  start_cost:
+    source_file: "files/model.yaml"
+    extraction_path: "model.generator.[name].startcost.value"
+    is_list: False
+    default_value: 0.0 # TODO: Check if this is correct
+    cast_type: float
+  generator_efficiency_curve:
+    is_subtype: True
+    cast_type: generator_efficiency_curve
+    is_list: False
+  turbine_efficiency_curves:
+    is_subtype: True
+    cast_type: turbine_efficiency_curve
+    is_list: True
+    source_file: "files/model.yaml"
+    extraction_path: "model.generator.[name].turb_eff_curves"
+
+generator_efficiency_curve:
+  power:
+    source_file: "files/model.yaml"
+    extraction_path: "model.generator.[name].gen_eff_curve.x"
+    is_list: True
+    default_value: [0.0] # TODO: Check if this is correct
+    cast_type: float
+  efficiency:
+    source_file: "files/model.yaml"
+    extraction_path: "model.generator.[name].gen_eff_curve.y"
+    is_list: True
+    default_value: [0.0] # TODO: Check if this is correct
+    cast_type: float
+
+turbine_efficiency_curve:
+  head:
+    source_file: "files/model.yaml"
+    extraction_path: "ref"
+    is_list: False
+    default_value: 0.0 # TODO: Check if this is correct
```

If we only have one turbine efficiency curve (defined for one head value), the head value does not matter
```suggestion
    default_value: 0.0
```